woocommerce/woocommerce-gutenberg-products-block
# Introduce view switcher component for the cart block

This PR introduces a `<ViewSwitchControl>` component which can be passed a list of views for display in a block.

As well as creating the component, this PR implements it within the cart block allowing the user to switch between "full" and "empty" cart views. State is managed by the cart block.

Fixes #1354



#### Accessibility


### Screenshots

![Screenshot 2020-01-15 at 16 14 16](https://user-images.githubusercontent.com/90977/72450336-18eb4680-37b2-11ea-9728-cf235e4b7bf3.png)

### How to test the changes in this Pull Request:

1. Add the cart block to a page
2. Click/focus on the block. Switcher should appear.
3. Try switching between the states. Also try clicking on inner blocks and confirm it stays visible.
